plans/main.fmf

```diff
@@ -0,0 +1,12 @@
+prepare:
+    - how: install
+      package:
+        - git-core
+        - golang
+        - rpmdevtools
+
+/podman_build_test:
+    summary: Build Podman
+    execute:
+        how: tmt
+        script: bash ./plans/podman_build_test.sh
```

Comment:
As it's not directly related to building, running, or documenting podman perhaps this could live under `./contrib/tmtplans` or similar?

Reply:
will try with this change.